app/services/moment_keywords.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class MomentKeywords
+  def initialize(moment)
+    @moment = moment
+    @moment_keywords = []
+  end
+
+  def extract_moment_keywords
```

Comment:
Not a blocker by any means, but the class name and method name here seem a little redundant. Could be worth exploring some other ways to use the class, such as:

```ruby
MomentKeywords.for(moment)

# Or perhaps something like...

MomentKeywords.new(moment).call
```